build.gradle.kts

```diff
@@ -33,7 +33,7 @@ tasks.withType<PatchPluginXmlTask> {
 intellij {
     version.set("2023.3.2")
     updateSinceUntilBuild.set(false)
-    plugins.set(listOf("IdeaVIM:2.8.0"))
+    plugins.set(listOf("IdeaVIM:2.10.0"))
```

Comment:
I get the following error when trying to build this locally:
```
Cannot resolve plugin 'IdeaVIM' in version '2.10.0'
```
Is this missing corresponding the "EAP" postfix or did you add this here for when `2.10.0` becomes available and you don't want this PR to be merged beforehand anyway?

Reply:
Apologies for the lack of description. Yes, this PR is intended for the future, when version 2.10 is released.

Reply:
Thanks for the explanation and your efforts :bow: